tac/project-updates/2025/2025-annual-Hiero.md

```diff
@@ -0,0 +1,82 @@
+Our community includes maintainers and contributors from different companies and even some that are not employed by a company.
+This group includes new contributors (committers and maintainers) as shown in the contributor list of https://github.com/hiero-ledger/hiero-solo-action, which is an example of a community driven project attracting new contributors from India and Africa.
+Another positive example for contributions is the YouTube tutorial series for our Python SDK that has been created by one of our community members: https://www.youtube.com/playlist?list=PL6uVtTx-oqY8V-HBfenONJrsA0Z8xJymX
+
+## Project Adoption
+
+The project announced the upcoming elections for two vital positions on the Hiero Technical Steering Committee (TSC): the End User Seat and the Contributor Seat.
+These roles are instrumental in guiding the technical direction and ensuring the diverse representation of our community within the Hiero project.
+
+This election will be held on May 7th until May 31st, 2025.
+
+At the moment, we continue to welcome and encorage nominations.
+
+Our current nominees can be viewed [here](https://github.com/hiero-ledger/governance/tree/main/elections/nominees/mar-2025-election)
+
+## Goals
+
+### Performance Against Prior Goals
+
+Our initial target was to move all Hedera repositories to Hiero.
+While we've mostly finished this transition, we've initiated the process of refactoring the code to a more vendor-agnostic and unbranded architecture.
+Our main focus is our SDKs in addition to discussing workflows, roadmaps, and patterns.
+Information and current state on the transition from Hedera can be found [here](https://github.com/hiero-ledger/hiero/blob/main/transition.md).
+Information about our work on making the SDKs vendor neutral can be found [here](https://github.com/hiero-ledger/hiero/blob/main/blog/2025-03-17-sdk-workflows.md).
+
+With all of this work in progress, we plan to release the initial minor versions of all projects in the near future.
+Next, we plan a major release that will contain all the work from moving the code to a more vendor neutral release.
+
+Another main goal is the onboarding of community projects from the Hiero ecosystem.
+At the moment, we moved 3 community projects to Hiero and a new proposal is in the pipeline.
+Information about the onboarding of new projects can be found [here](https://github.com/hiero-ledger/hiero/blob/main/community-transition.md).
+
+On the project's management front, the team is focusing on evaluating and analyzing key project health metrics that track our progress in fostering a thriving and diverse community.
+These metrics enable us to assess the project's performance by measuring contributions, code velocity, and issue response and resolution times.
+
+Our metrics report can be found [here](https://docs.google.com/document/d/e/2PACX-1vRxg-3wU-eg0EWEhAu9SRab_sXwJBy5YDQqoeV6BLq0FnVnoO4V41kPYEDYJovtns6tpq9gkaSaSzlO/pub).
```

Comment:
Are the LFX Insights metrics not enough? Is there something that the Heiro community would like to see added to LFX Insights? Should we have a sync with the LFX Insights team?